src/core/mesh.vert

```diff
@@ -55,14 +56,15 @@ void main()
     transform[2] = vec4(row1.z, row2.z, row3.z, 0.0);
     transform[3] = vec4(row1.w, row2.w, row3.w, 1.0);
     local2World *= transform;
-
 #ifdef USE_NORMALS
+    nor = mat3(transpose(inverse(local2World))) * normal;
     normalMat = mat3(transpose(inverse(local2World)));
 #endif
 #else
 #ifdef USE_NORMALS
     normalMat = mat3(normalMatrix);
 #endif
+    vec4 subt = vec4(0.0, 0.0, 1.0, 1.0);
```

Comment:
Why not support this for regular models (not instanced)?

Reply:
An option. Still learning about gl uniforms and attributes and wasn't sure if there was overhead.

Reply:
Uniforms are generally really cheap, so I think it makes sense to add this to regular models as well. However, I can also add it afterwards, no problem.